PsychSourceGL/Source/Common/IOPort/IOPort.c

```diff
@@ -350,9 +350,12 @@ PsychError IOPORTOpenSerialPort(void)
         "values 'paramValue' to specific serial port parameters 'paramName'. Not all "
         "parameters are supported by all operating systems, and all settings have reasonable "
         "defaults. Settings unknown to a specific operating system are ignored.\n"
+		"WARNING: Be meticulous when it comes to the spelling in the configuration string, "
```

Comment:
Indentation problem: Used tabs, but should use spaces, in this case 2*4 = 8 spaces from start of line, to indent 2 levels.

I'd drop the "WARNING:", doesn't add much to the sentence, but in the end up to you.

Other than that, it's good.

Reply:
Arggh. Fixed, but I am not sure what to do with the new commit.

Reply:
No problem, I will squash it into the previous one.
You can always do such things locally on your branch and the force-push to GitHub. This will update the associated pull request.

Anyhow, will pull now, and probably do a new PTB minor bug fix / improvement release soon. Your change won't take effect until a later release on Windows, macOS, as I'm currently away from my Windows and macOS build machines and can't rebuild the mex files for Windows and macOS.